automations/lifter.py

```diff
@@ -10,26 +10,43 @@ class LifterAutomation(StateMachine):
 
     @state(first=True, must_finish=True)
     def move_upper_scale(self):
-        self.lifter.move(self.lifter.UPPER_SCALE)
-        self.next_state("move_complete")
+        self.move_height = self.lifter.UPPER_SCALE
+        self.next_state("arms_out")
 
     @state(must_finish=True)
     def move_balanced_scale(self):
-        self.lifter.move(self.lifter.BALANCED_SCALE)
-        self.next_state("move_complete")
+        self.move_height = self.lifter.BALANCED_SCALE
+        self.next_state("arms_out")
 
     @state(must_finish=True)
     def move_lower_scale(self):
-        self.lifter.move(self.lifter.LOWER_SCALE)
-        self.next_state("move_complete")
+        self.move_height = self.lifter.LOWER_SCALE
+        self.next_state("arms_out")
 
     @state(must_finish=True)
     def move_switch(self):
-        self.lifter.move(self.lifter.SWITCH)
-        self.next_state("move_complete")
+        self.move_height = self.lifter.SWITCH
+        self.next_state("arms_out")
+
+    @timed_state(must_finish=True, duration=0.5, next_state='move')
+    def arms_out(self, initial_call):
+        self.intake.extend(True)
+        self.intake.rotate(-1)
+        self.intake.arms_out = True
```

Comment:
I believe this is an internal flag for firing the intake arm pistons at the start of a match, so there's no reason the lifting automation should be touching it…